src/acc/cuda_hip/acc_dev.cpp

```diff
@@ -49,10 +49,8 @@ extern "C" int c_dbcsr_acc_set_active_device(int device_id) {
   // establish context
   ACC_API_CALL(Free, (0));
 
-#if defined(__HIP_PLATFORM_NVCC__)
-  if (verbose_print) {
-    ACC_API_CALL(DeviceSetLimit, (ACC(LimitPrintfFifoSize), (size_t)1000000000));
-  }
+#if defined(__HIP_PLATFORM_NVCC__) && defined(ACC_DEV_PRINT_VERBOSE)
+  ACC_API_CALL(DeviceSetLimit, (ACC(LimitPrintfFifoSize), (size_t)1000000000));
```

Comment:
Here I will add a single:
```
#if defined(__CUDA) || defined(__HIP_PLATFORM_NVCC__)
ACC_API_CALL(DeviceSetLimit, (ACC(LimitPrintfFifoSize), (size_t)1000000000));
#endif
```

Reply:
Thanks, will do it accordingly! I simply overlooked the whole condition was meant to actually call this function (I was somehow convinced it's debug-only code).

Reply:
It comes to plan with only debugging, meaning to restrict the printf on the other kernels where we use the debugging (e.g. https://github.com/cp2k/dbcsr/blob/46c43eda9914f76cb059a816a9439a3eb651070b/src/acc/cuda_hip/acc_event.cpp#L22). Of course, we do set it to `0`.
I think the entire `verbose_print = 1` in this file is really misleading, so better to remove it. Thanks!